04-HTML5-APIs/Ex02/script.js

```diff
@@ -0,0 +1,34 @@
+function dragOver(e) {
```

Comment:
Let's assign those function to a variable in order to not populate the window scope.

`let dragOver = function (e) {}; `